app/services/sp_return_url_resolver.rb

```diff
@@ -11,25 +11,30 @@ def initialize(service_provider:, oidc_state: nil, oidc_redirect_uri: nil)
 
   # @return [String, nil]
   def return_to_sp_url
-    oidc_access_denied_redirect_url.presence ||
-      service_provider.return_to_sp_url.presence ||
-      inferred_redirect_url
+    url = oidc_access_denied_redirect_url.presence ||
+          service_provider.return_to_sp_url.presence ||
+          inferred_redirect_url
+    interpolate_url(url)
```

Comment:
I am not sure if this should apply to all of these URLs or if it should be restricted to just a few cases in this file.

Reply:
Specifically, for the SP we are putting this together for we only need to do the `post_idv_follow_up_url` so it may make sense to restrict things to just that case.

Reply:
i'd say let's restrict as much as possible, yeah

Reply:
I just pushed a change to only apply it to `post_idv_follow_up_url`